### PR TITLE
comment heavy playbook

### PR DESCRIPTION
**Pull request by bcoca**
based on pedantically commented playbook of old

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

examples

### CODE REVIEW

File: examples/play.yml [hunk 1]
```diff
@@ -0,0 +1,522 @@
...
+      when: '{{is_ubuntu|bool}}'
+
+    - name: "shutdown the if host is in the government"
+      command: /sbin/shutdown -t now
+      when: "{{inventory_hostname in groups['government']}}"
+
+      # another way to write the same.
+    - name: "shutdown the if host is in the government"
+      command: /sbin/shutdown -t now
+      when: "{{'goverment' in group_names}}"
```
Comment (flowerysong) [inline examples/play.yml]:
```suggestion
      when: "{{'government' in group_names}}"
```

File: examples/play.yml [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+
+    # Ansible has some built in variables, you can check them here (TODO url)
+    # inventory_hostname is the name of the current host the task is executing for (derived from the hosts: keyword)
+    # group_names has the list of groups the current host (inventory_hostname) is part of
+    # groups is a mapping of the inventory groups with the list of hosts that belong to them
+
+    ##########
+    # Run things as other users!
+    #
+    # Each task has an optional keywords that can control which
```
Comment (flowerysong) [inline examples/play.yml]:
```suggestion
    # Each task has optional keywords that control which
```

File: examples/play.yml [hunk 3]
```diff
@@ -0,0 +1,522 @@
...
+    # Run things locally!
+    #
+    # Each task can also be delegated to the control host
+    - name: create tempfile
+      local_action: shell dd if=/dev/urandom of=/tmp/random.txt count=100
+
+    # which is equivalent to the following
+    - name: create tempfile
+      shell: dd if=/dev/urandom of=/tmp/random.txt count=100
+      delegatge_to: localhost
```
Comment (flowerysong) [inline examples/play.yml]:
```suggestion
      delegate_to: localhost
```

File: examples/play.yml [hunk 4]
```diff
@@ -0,0 +1,522 @@
...
+         name: nginx
+         state: restarted
+
+    # this one can be called by name or via any entry in the listen keyword
+    - name: reddis restarter
+      service:
+         name: reddis
+         state: restarted
+      listen:
+        - Restart reddis
```
Comment (flowerysong) [inline examples/play.yml]:
```suggestion
    - name: redis restarter
      service:
         name: redis
         state: restarted
      listen:
        - Restart redis
```

File: examples/play.yml [hunk 5]
```diff
@@ -0,0 +1,522 @@
...
+# Blank lines are ignored, so can be used # to create spacing to your taste.
+
+# Note about YAML: like Python, cares about whitespace, it requires actual spaces, tabs won't work.
+# Indent consistently throughout.
+# Two-space or four space indents is what most users prefer, but do whatever you like.
+#
+# If you're new to YAML, keep in mind that YAML documents, like XML
+# documents, represent a tree-like structure of nodes and text. More
+# familiar with JSON?  Think of YAML as a strict and more flexible JSON
+# with fewer significant characters (e.g., :, "", {}, []).
```
Comment (flowerysong) [inline examples/play.yml]: This is confusingly worded. Both strict and more flexible? In what way does it have fewer significant characters?

Comment (bcoca) [reply examples/play.yml]: strict about spacing, more flexible about quoting:

``` {"this": "is valid json"} ```

```{ this: 'is valid for yaml, but not json'}```

but i'll rewrite, i was trying to copy/edit the original but did really bad job here

**Pull request merged**